packages/web/server/queries/complex/pools/market.ts

```diff
@@ -15,16 +14,16 @@ export type PoolMarketMetrics = Partial<{
   feesSpent7dUsd: PricePretty;
 }>;
 
-const metricPoolsCache = new LRUCache<string, CacheEntry>(DEFAULT_LRU_OPTIONS);
+const metricPoolsCache = new RemoteCache();
 /** Get a cached Map with pool IDs mapped to market metrics for that pool. */
 export function getCachedPoolMarketMetricsMap(): Promise<
   Map<string, PoolMarketMetrics>
 > {
   return cachified({
     cache: metricPoolsCache,
     key: "pools-metrics-map",
-    ttl: 1000 * 10, // 10 seconds
-    staleWhileRevalidate: 1000 * 25, // 25 seconds, edge function run time limit
+    ttl: 1000, // 1 min
```

Comment:
Did the units of this change? (10000 -> became 1000, but 10s -> 1min)

Reply:
I just fixed this with a later commit. LMK if this addresses you comment.